### DISCUSSION TITLE
Works well in 1440p - screenshots to compare with 2160p

### DISCUSSION THREAD

**Discussion by heroutm** [Blinue/Magpie#40] [General]
Hi,

it works well! Thank you! I'm really looking forward to next updates of this tool.

Check this in Plague Tale Innocence game. It's a bit more blurry in 1440p with FSR, but it still looks better than native 1440p.  Performance went UP!

**2160p Native**

![2160p Native](https://user-images.githubusercontent.com/38924821/128522816-529a4892-12b6-49cd-89c2-c7c90b272f26.jpg)

**1440p FSR**

![1440p FSR](https://user-images.githubusercontent.com/38924821/128522824-3169780e-917b-4b0b-84c2-0a4e0c2b37a0.jpg)

**No problem in Cyberpunk at all.** 

I still think that 1440p with FSR looks better and sharper than native 2160p. At least for the first impression.
Look at this! I cannot believe it. Graffiti is a lot sharper, red tube on the ground (in the middle) has many more details than in 1440p native.  

**2160p Native**
![2160p Native](https://user-images.githubusercontent.com/38924821/128546888-68ac3746-f47b-4176-b82c-c7b13a756ee0.jpg)

**1440p FSR**
![1440p FSR](https://user-images.githubusercontent.com/38924821/128546881-68d7e245-cb3d-4ad6-8900-85b2c45deed9.jpg)

**1440p Native**

![1440p Native](https://user-images.githubusercontent.com/38924821/128546885-86b5099a-b635-4bc2-9faa-efa4b5c54bc9.jpg)

**Comment by Blinue** [comment]
It seems that the picture has turned yellow, what's going on?

**Reply by heroutm** [reply]
You mean the yellow edge or yellowish tones?

**Reply by Blinue** [reply]
both

**Reply by Blinue** [reply]
The yellow border should not appear because it has been cropped.

**Reply by heroutm** [reply]
By the way, I have Radeon Image Sharpening enabled in my drivers at 80% by default, but I'm not sure if this have something to do with it. Yellow edges are show when I press ALT + F11.

It's happening only when Magpie is running with WinRT Capture. 

You can also see a bit higher CPU usage, CPU Power draw and temperature when FSR is enabled.

**Reply by Blinue** [reply]
One drawback of WinRT capture mode is that it will take up certain CPU resources. I will try to remove the yellow border.

**Reply by heroutm** [reply]
Hmm, sometimes it shows just for a while when I activate FSR and then it disappears. CPU is okay by now, It must have been caused by something different. But when I change in game resolution and deactivate/activate again, it stays ON. I will test with other games and let you know.

**Reply by heroutm** [reply]
I can confirm that other games are not suffering from yellowish tint or yellow border at all.

**Reply by Blinue** [reply]
So this game is special, thanks for your feedback.

I think I've found a way to hide the yellow border. Although most people don't use it, I will provide this feature in the next version.
